.gitignore

```diff
@@ -30,6 +30,7 @@ MANIFEST
 # Environments
 env/
 venv/
+.venv/
```

Comment:
Surprisingy, this was not yet contained in the `.gitignore`. No idea why my machine ignored this in the past (I didn't have it in my `~/.gitignore`).

I need this for the configuration of yamlfmt.